File: Practica-cpp/Trabajo practicos/3-Estructuras Condicionales/Ejercicio_1.cpp
/*
 * 1. Escriba un programa que lea dos numeros y determine cual de ellos es el mayor
 */

#include <iostream>
using namespace std;
int main() {
    int numero1, numero2;

    cout<<"Ingrese un numero a: ";cin>>numero1;
    cout<<"Ingrese un numero b: ";cin>>numero2;

    if (numero1 > numero2) {
        cout<<"El numero "<<numero1<<" es mayor que "<<numero2<<endl;
    }else if (numero2==numero1) {
        cout << "Los dos Numeros Son Iguales " <<endl;
    }else {
        cout<<"El numero "<<numero2<<" es mayor que "<<numero1<<endl;
    }
    return 0;
}

/*
 * 2. Escriba un programa que lea tres numeros y determine cual de ellos es el mayor
 */

File: Practica-java/src/Trabajo_Practico_1/_3_Estructuras_Condicionales/Ejercicio_1.java
package Trabajo_Practico_1._3_Estructuras_Condicionales;
import java.util.Scanner;
public class Ejercicio_1 {
    public static void main(String[] arg){
        Scanner entrada = new Scanner(System.in);
        int numero1, numero2;

        System.out.println("Ingrese un numero a: ");
        numero1=entrada.nextInt();
        System.out.println("Ingrese un numero b: ");
        numero2=entrada.nextInt();

        if (numero1 > numero2) {
            System.out.println("El numero "+numero1+" es mayor que "+numero2);
        }else if (numero2==numero1) {
            System.out.println("Los dos Numeros Son Iguales ");
        }else {
            System.out.println("El numero "+numero2+" es mayor que "+numero1);
        }

    }
}
